src/_cffi_src/openssl/aes.py

```diff
@@ -26,22 +26,22 @@
                  const unsigned char *, unsigned int);
 int AES_unwrap_key(AES_KEY *, const unsigned char *, unsigned char *,
                    const unsigned char *, unsigned int);
+"""
+
+MACROS = """
 /* The ctr128_encrypt function is only useful in 1.0.0. We can use EVP for
    this in 1.0.1+. */
 void AES_ctr128_encrypt(const unsigned char *, unsigned char *,
                         size_t, const AES_KEY *, unsigned char[],
                         unsigned char[], unsigned int *);
 """
 
-MACROS = """
-"""
-
 CUSTOMIZATIONS = """
 static const long Cryptography_HAS_AES_WRAP = 1;
 #if OPENSSL_VERSION_NUMBER >= 0x10100000L && !defined(LIBRESSL_VERSION_NUMBER)
```

Comment:
Now that I look at this, can you explain the `OPENSSL_VERSION_NUMBER` check here?